### DISCUSSION TITLE
Error after complete run enlightn and trigger enlightn bot. is it normal?

### DISCUSSION THREAD

**Discussion by asfarul** [enlightn/enlightn#74] [Q&A]
Hello, 
i got an error while running workflow enlightn. but it looks no error, but why the end of process i got an error? 
![Screenshot_39](https://user-images.githubusercontent.com/13598088/130913258-8bb7c950-c2e0-41dd-a1e5-3824f56c4c17.png)

thanks before :)

**Comment by perisicnikola37** [comment]
Same situation but that is happening to you because there are some errors in your tests

**Comment by paras-malhotra** [comment]
This is normal / fine. Enlightn exits with a non-zero code when there are some failed checks for easy CI.

Here's the code in reference: https://github.com/enlightn/enlightn/blob/8c2ea190403ed9283ba7efce72195b35c49c4131/src/Console/EnlightnCommand.php#L129-L132
